src/file-upload/file-upload.scss

```diff
@@ -39,8 +39,9 @@
       z-index: -1;
     }
 
-    > svg {
+    svg {
```

Comment:
sure no class? We now apply on all the svgs under content...
Also, need to think on default display for our icons mixin, cause now it seems we cannot apply them anywhere on svgs we want..

Reply:
If we add a class, then the user will need to specify it manually.

Maybe I'll revert to `> svg`.